src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1134,6 +1134,7 @@ public void testContainerWithHostConfig() throws Exception {
         .publishAllPorts(publishAllPorts)
         .dns(dns)
         .cpuShares((long) 4096)
+        .cpuQuota((long) 50000)
```

Comment:
I added the assertion. On my local machine test passes, while on CircleCI I have a failure...